app/components/dns-record-table-row.tsx

```diff
@@ -76,30 +76,56 @@ export default function DnsRecordsTableRow({
               {dnsRecord.expiresAt.toLocaleDateString('en-US')}
             </Text>
           </Show>
-          <ButtonGroup spacing={{ base: 'auto', sm: '0.5' }} width="full">
```

Comment:
Is the only difference between the blocks inside the Hide and Show the styles on the ButtonGroup. That is, one has `spacing={{ base: 'auto', sm: '0.5' }} width="full"`, and the other doesn't ?

Reply:
Yes.

Reply:
Instead of using Hide and Show for this, you can do this: 
<img width="700" alt="image" src="https://user-images.githubusercontent.com/67077705/234993236-291c7941-66ee-42fc-9cfb-61e52cd9f41f.png">
Seems to work the same from my testing

Reply:
I didn't realize that the width would be the one causing the issue